docs/browser_support.md

```diff
@@ -0,0 +1,18 @@
+# Whitehall browser support
+
+Whitehall browser support follows the guidance provided in the [GOV.UK service manual](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in-from-june-2022).
+
+
+## Internet Explorer 11 support
+
+Whitehall's support of Internet Explorer 11 does not go beyond the functionality that is offered through progressive enhancement. Javascript is not required to be compatible with the browser but should not cause the page to break or  become non-functional.
+
+If a module uses features such as `fetch`, the presence of these features should be checked in the initialisation method before any changes are made to the DOM.
+
+## Usage of ES6 (2015) syntax
+
+To enable the use of ES6 syntax such as arrow functions Whitehall's build process is in the process of being updated. Upon completion of this work, Javascript will be entirely disabled in Internet Explorer 11 through the use of modules. Only following this variable declarations such as `let` and `const` be used.
```

Comment:
```suggestion
To enable the use of ES6 syntax such as arrow functions Whitehall's build process is in the process of being updated. Upon completion of this work, JavaScript will be entirely disabled in Internet Explorer 11 through the use of modules. Only following this can variable declarations such as `let` and `const` be used.
```